packages/core/src/tegel-light/components/tl-toggle/tl-toggle.scss

```diff
@@ -0,0 +1,159 @@
+@import '../../../../../../typography/utilities/typography-utility';
+@import '../../../mixins/focus-state';
+@import './tl-toggle-vars';
+
+.tl-toggle {
+  .disabled {
+    pointer-events: none;
+  }
+
+  .toggle-headline {
+    @include detail-02;
+
+    color: var(--toggle-text);
+    margin-bottom: 12px;
+
+    &.disabled {
+      color: var(--toggle-disabled);
+      user-select: none;
+    }
+  }
+
+  input[type='checkbox'] {
+    appearance: initial;
+    margin: 0;
+    width: 44px;
+    height: 24px;
+    position: relative;
+    display: inline-block;
+    vertical-align: middle;
+    cursor: pointer;
+
+    &::after,
+    &::before {
+      transition: all var(--tds-motion-duration-moderate-01) var(--tds-motion-easing-scania);
+      content: '';
+      position: absolute;
+      box-sizing: border-box;
+    }
+
+    &::before {
+      /* Slider */
+      background-color: var(--toggle-background-off);
+      border: 2px solid transparent;
+      border-radius: 16px;
+      left: 0;
+      height: 24px;
+      width: 44px;
+    }
+
+    /* Switch */
+    &::after {
+      background-color: var(--toggle-switch);
+      border-radius: 50%;
+      left: 4px;
+      top: 4px;
+      width: 16px;
+      height: 16px;
+    }
+
+    &:focus {
+      @include tds-focus-state;
+
+      border-radius: 16px;
+
+      &::before {
+        background-color: var(--toggle-disabled);
+      }
+    }
```

Comment:
Here do `:focus-visible` so we only see the focus ring when tabbing to the toggle component.